src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_account_scenarios.py

```diff
@@ -1029,3 +1029,19 @@ def test_storage_account_failover(self, resource_group):
             self.check('name', '{sa}'),
             self.check('failoverInProgress', True)
         ])
+
+
+class StorageAccountLocalContextScenarioTest(LocalContextScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='clistorage', location='westus2')
+    def test_storage_account_local_context(self):
```

Comment:
Do we need to turn on local context first?
Do we need to check location?

Reply:
Turn on local context is already included in the base class. For location it is similar to resource_group_name, as there are too many commands which has location, I think we do not need to check for it one by one.